spec/factories/miq_region.rb

```diff
@@ -1,5 +1,7 @@
 FactoryGirl.define do
+  my_region_number = MiqRegion.my_region_number
+
   factory :miq_region do
```

Comment:
Maybe this should have a different name like `region_remote` if the intention is that it will never be the local region?